data_processors/pipeline/lambdas/umccrise.py

```diff
@@ -0,0 +1,177 @@
+from data_processors.pipeline.services import sequence_run_srv, batch_srv, workflow_srv, metadata_srv, library_run_srv
+from data_processors.pipeline.domain.workflow import WorkflowType, SecondaryAnalysisHelper
+from data_processors.pipeline.lambdas import wes_handler
+from data_processors.pipeline.tools.liborca import get_tiny_uuid
+
+from utils import libjson, libssm, libdt
+
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
+
+
+def sqs_handler(event, context):
+    """event payload dict
+    {
+        'Records': [
+            {
+                'messageId': "11d6ee51-4cc7-4302-9e22-7cd8afdaadf5",
+                'body': "{\"JSON\": \"Formatted Message\"}",
+                'messageAttributes': {},
+                'md5OfBody': "e4e68fb7bd0e697a0ae8f1bb342846b3",
+                'eventSource': "aws:sqs",
+                'eventSourceARN': "arn:aws:sqs:us-east-2:123456789012:fifo.fifo",
+            },
+            ...
+        ]
+    }
+
+    Details event payload dict refer to https://docs.aws.amazon.com/lambda/latest/dg/with-sqs.html
+    Backing queue is FIFO queue and, guaranteed delivery-once, no duplication.
+
+    :param event:
+    :param context:
+    :return:
+    """
+    messages = event['Records']
+
+    results = []
+    for message in messages:
+        job = libjson.loads(message['body'])
+        results.append(handler(job, context))
+
+    return {
+        'results': results
+    }
+
+
+def handler(event, context) -> dict:
+    """event payload dict
+    {
+        "dragen_somatic_directory": {
+            "class": "Directory",
+            "location": "gds://path/to/somatic/output/dir"
+        },
+        "fastq_list_rows_germline": [{
+            "rgid": "index1.index2.lane",
+            "rgsm": "sample_name",
+            "rglb": "UnknownLibrary",
+            "lane": int,
+            "read_1": {
+              "class": "File",
+              "location": "gds://path/to/read_1.fastq.gz"
+            },
+            "read_2": {
+              "class": "File",
+              "location": "gds://path/to/read_2.fastq.gz"
+            }
+        }],
+        "output_directory_germline": "PRJ1234567",
+        "output_directory_umccrise": "L2100123_L2200124",
+        "output_file_prefix_germline": "PRJ1234567"
+        "seq_run_id": "sequence run id",
+        "seq_name": "sequence run name",
+        "batch_run_id": "batch run id",
+        "library_id": "library id",
+    }
+
+    :param event:
+    :param context:
+    :return: workflow db record id, wfr_id, sample_name in JSON string
+    """
+
+    logger.info(f"Start processing {WorkflowType.UMCCRISE.name} event")
+    logger.info(libjson.dumps(event))
+
+    seq_run_id = event.get('seq_run_id', None)
+    seq_name = event.get('seq_name', None)
+    batch_run_id = event.get('batch_run_id', None)
+    library_id = event['library_id']
+
+    # Set workflow helper
+    wfl_helper = SecondaryAnalysisHelper(WorkflowType.UMCCRISE)
```

Comment:
Just to note myself; need to work on workflow helper here a bit. It seems using older interface or methods. No worries about it, I will take care of it!